dev/PowerNotifications/PowerNotifications.h

```diff
@@ -608,6 +608,11 @@ namespace winrt::Microsoft::ProjectReunion::factory_implementation
         //SystemSuspend Functions
         winrt::Microsoft::ProjectReunion::SystemSuspendStatus SystemSuspendStatus()
         {
+Addi            // Review: Agree with this usage-only-after-subscription enforcement?
+            if (m_systemSuspendStatus == SystemSuspendStatus::Uninitialized)
+            {
+                throw winrt::hresult_error(E_FAIL, L"There are no active subscriptions on SystemSuspendStatus");
```

Comment:
I think "Only callable during SystemSuspendStatusChanged callback" is more accurate.

Reply:
I was referring to the error message on line 614. Even if you have an active subscription, you're blocked from calling this method from outside of a callback, right? That's why I'm recommending that the message be clarified.